SimplenoteUITests/NotesList.swift

```diff
@@ -6,12 +6,24 @@ class NoteList {
         app.tables.cells[noteName].tap()
     }
 
+    class func getNavBarIdentifier() -> String {
+        let navBar = app.navigationBars.element
+        guard navBar.exists else { return "" }
+        print(">>> Currently active navigation bar: " + navBar.identifier)
+        return navBar.identifier
+    }
+
     class func isAllNotesListOpen() -> Bool {
         return app.navigationBars[UID.NavBar.allNotes].exists
     }
 
+    class func isNoteListForTagOpen(tag: String) -> Bool {
```

Comment:
Nitpick. Swift allows us to have two different argument labels, one for the call site and to use internally. This helps write code that reads more like English:

```suggestion
    class func isNoteListOpen(forTag tag: String) -> Bool {
```

This code still uses `tag` internally, but is called like `isNoteListOpen(forTag: "a-tag")` which reads more like a sentence, imho.

Reply:
Thank you! Used your suggestion, and currently the impact (changes required I mean) was 0, because the method was never called. It's more for the future.